gravitee-node-certificates/pom.xml

```diff
@@ -77,12 +77,12 @@
         <!-- Bouncy Castle -->
         <dependency>
             <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk15on</artifactId>
+            <artifactId>bcprov-jdk18on</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
-            <artifactId>bcpkix-jdk15on</artifactId>
+            <artifactId>bcpkix-jdk18on</artifactId>
```

Comment:
While doing this task I noticed that this lib bcpkix-jdk15on was also moved to bcpkix-jdk18on so I bumped it up as well. I hope that's fine

Reply:
Hey, It's fine, but you need to create a PR in APIM with 5.8.2-apim-4028-update-to-bcprov-jdk18on-SNAPSHOT version of node and make sure all tests pass. I'm pretty sure it is used in the project (look at gravitee-gateway-test-sdk and integration-tests projects first.

Reply:
pom.xml
gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-container/pom.xml
gravitee-apim-rest-api/gravitee-apim-rest-api-idp/gravitee-apim-rest-api-idp-core/pom.xml
gravitee-apim-gateway/gravitee-apim-gateway-security/gravitee-apim-gateway-security-jwt/pom.xml
gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/pom.xml

Reply:
Those files make ref to BC. Make sure it is up to date.

Reply:
Thanks @benoitgravitee. Sure thing, here I created APIM PR with the changes you mentioned: 
https://github.com/gravitee-io/gravitee-api-management/pull/6837/files